modules/op-release-notes-1-19.adoc

```diff
@@ -0,0 +1,396 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-19.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-19_{context}"]
+= Release notes for {pipelines-title} General Availability 1.19
+
+With this update, {pipelines-title} General Availability (GA) 1.19 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-19_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.19:
+
+[id="pipelines-new-features-1-19_{context}"]
+=== Pipelines
+
+* With this update, you can now specify custom `securityContext` settings under the `EventListener` YAML specification. When custom `securityContext` is enabled, user-defined values override the default configuration. If no custom values are provided, the default `securityContext` setting is applied automatically.
++
+.Example configuration for enabling custom `securityContext` in the `EventListener` YAML spec
+[source,yaml]
+----
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  resources:
+    kubernetesResource:
+      spec:
+        template:
+          spec:
+            securityContext:
+              runAsNonRoot: true
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+              securityContext:
+                readOnlyRootFilesystem: true
+----
+
+* With this update, you can configure custom database credentials for {tekton-results} using the `TektonConfig` custom resource (CR). This eliminates the need to rely on default PostgreSQL secrets that use default usernames and passwords.
+
+* With this update, {pac} no longer creates a pending status on GitHub Pull Requests when an unauthorized bot user attempts to trigger a `PipelineRun`. Instead of generating a blocking status check, such requests are now silently disallowed.
+
+* With this update, a new metric, `pipelines_as_code_git_provider_api_request_count`, tracks the number of API calls made by {pac} to Git providers, such as GitHub, GitLab, and Gitea. The metric also helps to monitor API rate limit usage across Git providers, namespaces, event types, and repositories.
```

Comment:
```suggestion
* With this update, a new metric, `pipelines_as_code_git_provider_api_request_count`, tracks the number of API calls made by {pac} to Git providers, such as GitHub, GitLab, and Gitea. The metric also helps to monitor API rate limit usage per Git provider, namespace, event type, and repository.
```

Not sure if my suggestion is preferable or not, but the metric is tagged with the api request's git provider, namespace, etc. My reading of the original wording made me think it would not be tagged with those facets. I'm not an expert in technical writing though; if the original wording is appropriate then it is fine. I just wanted to make sure it accounted for the tags.

Reply:
updated on google doc

Reply:
Fixed.